from userver.service import main
try:
    main()
except Exception as e:
    print(e)
